docs/notes-KZG10.md

```diff
@@ -1,3 +1,53 @@
 In this module we show how and why the KZG10 
 polynomial commitment scheme has been modularised 
```

Comment:
Maybe modularised is not the term. Maybe `added` or `implemented` would fit better

Reply:
I'll just call it implemented.